qemu/tests/drive_mirror.py

```diff
@@ -149,6 +148,14 @@ def clean(self):
             image = nfs.Nfs(params)
             image.cleanup()
 
+    def create_files(self):
```

Comment:
Maybe you should consider to move these two method into the parent class.
They are defined in  #1017  as well

Reply:
@hereischen Yes, I will move it. Thanks.